packages/sit-onyx/src/components/OnyxDataGrid/OnyxDataGrid.stories.ts

```diff
@@ -0,0 +1,21 @@
+import type { Meta, StoryObj } from "@storybook/vue3";
+import OnyxDataGrid from "./OnyxDataGrid.vue";
+
+const meta: Meta<typeof OnyxDataGrid> = {
+  title: "DataGrid/DataGrid",
+  component: OnyxDataGrid,
+};
+
+export default meta;
+type Story = StoryObj<typeof OnyxDataGrid>;
+
+export const Default = {
+  args: {
+    columns: ["name", "age", "birthday"],
+    data: [
+      { id: 1, name: "Alice", age: 30, birthday: new Date("1990-01-01") },
+      { id: 2, name: "Bob", age: 25, birthday: new Date("1995-06-15") },
+      { id: 3, name: "Charlie", age: 35, birthday: new Date("1998-02-11") },
```

Comment:
```suggestion
      { id: 1, name: "Alice", age: 30, birthday: new Date("1990-01-01") },
      { id: 2, name: "Charlie", age: 35, birthday: new Date("1998-02-11") },
      { id: 3, name: "Bob", age: 25, birthday: new Date("1995-06-15") },
```

We could make the default order unsorted so the user can notice a difference between no sorting and ascending